lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -90,12 +90,13 @@ export default function FilePickerApp({ onSubmit }) {
   const submitButton = /** @type {{ current: HTMLInputElement }} */ (useRef());
   const {
     api: { authToken },
-    filePicker: { deepLinkingAPI, formAction, formFields, blackboard, canvas },
+    product: { settings },
```

Comment:
Is it possible to destructure `settings.groupsEnable` directly here as 
`enableGroupConfig`?

Reply:
It is possible with `product: { settings: { groupsEnable: enableGroupConfig } }`. You might find it more readable to do the destructuring in multiple stages though, ie do one or two levels in the first statement, then the rest in a separate statement.